pkcs11/src/new/types/object.rs

```diff
@@ -140,23 +193,34 @@ impl Attribute<'_> {
             | Attribute::Verify(_)
             | Attribute::VerifyRecover(_)
             | Attribute::Wrap(_) => std::mem::size_of::<CK_BBOOL>(),
-            Attribute::Base(_) => std::mem::size_of::<CK_BYTE>(),
```

Comment:
The PKCS11 specification says that `CK_BYTE` is one byte so I think it's fine to replace here.